libcasr/src/gdb/mod.rs

```diff
@@ -21,6 +21,10 @@ impl ParseStacktrace for GdbStacktrace {
             .collect())
     }
 
+    fn parse_stacktrace_entry(entry: &str) -> Result<StacktraceEntry> {
+        Ok(StacktraceEntry::new(entry)?)
+    }
+
     fn parse_stacktrace(entries: &[String]) -> Result<Stacktrace> {
```

Comment:
Do we need this? We already have a default implementation.

Reply:
I checked `from_gdb` implementation from `gdb-command` and noticed that it does additional work like stripping lines, skipping empty ones, etc.